subsys/bluetooth/audio/ascs.c

```diff
@@ -1446,6 +1508,33 @@ static ssize_t ascs_config(struct bt_ascs *ascs, struct net_buf_simple *buf)
 	return buf->size;
 }
 
+int bt_audio_ep_get_info(const struct bt_audio_ep *ep, struct bt_audio_ep_info *info)
+{
+	info->id = ep->status.id;
+	info->state = ep->status.state;
+	info->dir = ep->dir;
+
+	return 0;
+}
+
+void bt_audio_unicast_server_foreach_ep(struct bt_conn *conn,
```

Comment:
This function should probably be put in `unicast_server.c` instead.

Reply:
In `bt_audio_unicast_server_foreach_ep()` there is usage of static `ascs_get()`. Can we leave the function here or is it better to create non-static version of `ascs_get()`?

Reply:
I would suggest to move `ascs_get()` to `ascs_internal.h` and rename to `bt_ascs_get`

Reply:
I've faced a problem with circular imports when moving all contents of `bt_audio_unicast_server_foreach_ep` and `bt_audio_unicast_server_config_ase` into `unicast_server.c`. To fix this, the `struct bt_ascs_ase` and `struct bt_ascs` would have to be moved into `endpoint.h`. Finally I would need `bt_ascs_get`, `bt_ascs_find` and `bt_ascs_new` in `ascs_internal.h`...

So to avoid major changes, in `ascs_internal.h` there are now:
```C
int bt_ascs_config_ase(struct bt_conn *conn, struct bt_audio_stream *stream, struct bt_codec *codec,
		       const struct bt_codec_qos_pref *qos_pref);

void bt_ascs_foreach_ep(struct bt_conn *conn, bt_audio_ep_func_t func, void *user_data);
```
which are called from `bt_audio_unicast_server_config_ase` and `bt_audio_unicast_server_foreach_ep` in `unicast_server.c`. That way we would keep the naming convention and leave room for the future refactors